src/main/java/org/springframework/cloud/deployer/spi/mesos/chronos/ChronosTaskLauncher.java

```diff
@@ -95,7 +95,7 @@ public String launch(AppDeploymentRequest request) {
 			job.setArguments(args);
 		}
 		job.setSchedule("R1//P");
-		job.setRetries(1);
+		job.setRetries(properties.getRetries());
```

Comment:
Requires update to copyright year 2017 in both the files. will fix it when merging.